tests/runtime/probe

```diff
@@ -30,6 +30,13 @@ EXPECT SUCCESS [0-9][0-9]*
 TIMEOUT 5
 AFTER ./testprogs/syscall read
 
+# https://github.com/iovisor/bpftrace/issues/2447
+NAME kfunc_multiple_attach_point_multiple_functions
+PROG BEGIN { @a = 2; @b = 1; @c = 1 } i:s:1 { exit() } kfunc:vfs_read, kfunc:vfs_open /@a != 0/ { @a -= 1 } kfunc:vfs_read /@b == 1/ { @b = 0 } kfunc:vfs_open /@c == 1/ { @c = 0 }
+EXPECT @a: 0\n@b: 0\n@c: 0
+TIMEOUT 5
```

Comment:
```suggestion
TIMEOUT 5
REQUIRES_FEATURE btf
```

Reply:
I wonder why the other test cases (e.g. the one I copied mine from) don't have this, I thought kfunc usually (always?) require BTF?

Reply:
It does, if some test is using kfuncs and missing the BTF requirement, it's a mistake.